report/src/sections/01-introduction.tex

```diff
@@ -1,22 +1,28 @@
 \section{Introduction}\label{sec:introduction}
-\IEEEPARstart{S}{oftware} analysis is a broad field encompassing a wide range of techniques and methodologies, each tailored to address specific aspects of software systems.
-One such technique is value analysis, which focuses on the behavior of a program with respect to its input and output values~\cite{jackson_software_2000}.
-Value analysis can be approached from various angles, including the consideration of meta-information like information flow or taint analysis.
-While these approaches offer valuable insights into the behavior of a program, they often require detailed knowledge of the program's internal workings and are not always applicable to all types of software systems.
-In this paper we use static program analysis, specifically abstract interpretation.
+\IEEEPARstart{S}{oftware} analysis is a broad field encompassing various techniques and methodologies, each designed to address specific aspects of software systems.
+Among these techniques, data flow analysis is a fundamental method for systematically analyzing software behaviour.
+Within data flow analysis, abstract interpretation emerges as a powerful tool for examining software behaviour through static analysis.
```

Comment:
Abstract interpretation is not within data flow analysis, neither is it a subset.